src/khoj/processor/content/markdown/markdown_to_entries.py

```diff
@@ -114,7 +114,7 @@ def convert_markdown_entries_to_maps(parsed_entries: List[str], entry_to_file_ma
             # Append base filename to compiled entry for context to model
             # Increment heading level for heading entries and make filename as its top level heading
             prefix = f"# {stem}\n#" if heading else f"# {stem}\n"
-            compiled_entry = f"{prefix}{parsed_entry}"
+            compiled_entry = f"{entry_filename}\n{prefix}{parsed_entry}"
```

Comment:
Do we need to do this here? Seems like the `entry_filename` is being used via `prefix` -> `stem` -> `Path(raw_filename).stem`

Reply:
that would get only the file name, but not the whole path. i think this loses important metadata if not included (e.g., look at the research paper in my downloads folder about orangutans).